src/runtime-rs/crates/hypervisor/src/lib.rs

```diff
@@ -26,6 +26,13 @@ use kata_types::config::hypervisor::Hypervisor as HypervisorConfig;
 // Config which driver to use as vm root dev
 const VM_ROOTFS_DRIVER_BLK: &str = "virtio-blk";
 const VM_ROOTFS_DRIVER_PMEM: &str = "virtio-pmem";
+// before using hugepages for VM, we need to mount hugetlbfs
+// /dev/hugepages will be the mount point
+// mkdir -p /dev/hugepages
+// mount -t hugetlbfs none /dev/hugepages
+const DEV_HUGEPAGES: &str = "/dev/hugepages";
```

Comment:
You assume the mount point could only be `/dev/hugepages` here. Could this be configurable?

Is this behaviour the same in Go runtime?

Reply:
Yes, you can refer to this https://github.com/kata-containers/kata-containers/blob/main/src/runtime/pkg/govmm/qemu/qemu.go#L2848